server/index.js

```diff
@@ -1,11 +1,16 @@
 const { Configuration, OpenAIApi } = require("openai");
 const express = require("express");
 const cors = require("cors");
+
+require('dotenv').config
+
 const app = express();
-const PORT = 4000;
+const PORT = process.env.PORT;
```

Comment:
Can you add an .env.example file that has these variables in there?

Reply:
Sure, i will added it right away

Reply:
Added